pkg/trace/traceutil/span.go

```diff
@@ -8,23 +8,21 @@ package traceutil
 import "github.com/DataDog/datadog-agent/pkg/trace/pb"
 
 const (
-	// TraceMetricsKey is a tag key which, if set to true,
-	// ensures all statistics are computed for this span.
-	// [FIXME] *not implemented yet*
-	TraceMetricsKey = "datadog.trace_metrics"
-
 	// This is a special metric, it's 1 if the span is top-level, 0 if not.
 	topLevelKey = "_top_level"
+
+	// This is a special meta tag that marks a span for metrics calculation.
+	measuredKey = "_dd.measured"
```

Comment:
Do we want to make this an exported var?

Reply:
No, this is fine.